sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -84,6 +85,30 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase {
       result.foreach { case Row(_, _, _, info: String) => assert(info.nonEmpty) }
     }
   }
+
+  test("case sensitivity of partition spec") {
+    withNamespace(s"$catalog.ns") {
+      sql(s"CREATE NAMESPACE $catalog.ns")
+      val t = s"$catalog.ns.part_table"
+      withTable(t) {
+        sql(s"""
+          |CREATE TABLE $t (price int, qty int, year int, month int)
+          |$defaultUsing
+          |partitioned by (year, month)""".stripMargin)
```

Comment:
nit. `partitioned by` -> `PARTITIONED BY` (upper case) because we used `CREATE TABLE` in the upper cases.

Reply:
I am working on V2 SHOW TABLE EXTENDED: https://issues.apache.org/jira/browse/SPARK-33393 at the moment. When I will be moving the test to the common test suite to run it for both V1 and V2, I will address your comment.